### ISSUE TITLE
omit_parameter_name, not reported for builtin?

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#2800]
expect finding for,

```abap
DATA foo TYPE string.
foo = 'hello'.
foo = to_upper( val = foo ).
```

![image](https://user-images.githubusercontent.com/5888506/204575316-83b56c4b-c294-4742-8db6-084c65a465cd.png)
